src/Jobs/CheckLink.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Portable\FilaCms\Jobs;
+
+use Illuminate\Bus\Queueable;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Foundation\Bus\Dispatchable;
+use Illuminate\Queue\InteractsWithQueue;
+use Illuminate\Queue\SerializesModels;
+use Portable\FilaCms\Models\LinkCheck;
+use Illuminate\Support\Facades\Http;
+use GuzzleHttp\TransferStats;
+
+class CheckLink implements ShouldQueue
+{
+    use Dispatchable;
+    use InteractsWithQueue;
+    use Queueable;
+    use SerializesModels;
+
+    /**
+     * Create a new job instance.
+     */
+    public function __construct(
+        public LinkCheck $linkCheck,
+    ) {
+    }
+
+    public function handle(): void
```

Comment:
I would make it so that the jobs handle all of the getting of the links as well.